docs/cep/query-guide/functions/geospatial/crosses.md

```diff
@@ -33,7 +33,7 @@ This query uses the `geo:crosses()` function to determine if a line, defined by
 
 ```sql
 CREATE STREAM InputStream (lineStart double, lineEnd double);
-CREATE STREAM OutputStream (crosses bool);
+CREATE SINK STREAM OutputStream (crosses bool);
```

Comment:
@diana-macrometa, the  example is incorrect,
An example for example:
```
CREATE STREAM dataInForGeoCrossesStream (id string, longitude double, latitude double, timestamp long);
CREATE SINK STREAM dataOutForGeoCrossesStream (id string, crosses  bool);
CREATE TRIGGER GeoCrossesTrigger WITH(interval = 5 sec);

INSERT INTO dataInForGeoCrossesStream
SELECT  "12" AS id, 6.876657 AS longitude, 79.897648 AS latitude,  eventTimestamp() AS timestamp
from GeoCrossesTrigger;

@info(name = 'geoCrossesQuery')
INSERT INTO dataOutForGeoCrossesStream
SELECT "id" AS id, crosses AS crosses
FROM dataInForGeoCrossesStream#geo:crosses("12", 0.5, 0.5,"{'type':'Polygon','coordinates':[[[0,0],[0,2],[1,2],[1,0],[0,0]]]}");
```
the first parameter "id" needs to be a static string it seems can't be dynamic and this is a stream processor so it needs to be InputStream#geo:crosses..
maybe @StoyanTc has a better example for can explain more